src/core/device.c

```diff
@@ -716,10 +716,10 @@ static bool device_is_ready(sd_device *dev) {
         r = device_is_renaming(dev);
         if (r < 0)
                 log_device_error_errno(dev, r, "Failed to check if device is renaming: %m");
-        if (r > 0) {
+        if (r > 0)
                 log_device_debug(dev, "Device busy: renaming");
+        if (r != 0)
```

Comment:
This changes the previous behavior... Why do you change it??

Reply:
I figured it'd be good to not consider a device ready if an unexpected error happens, but if it might cause issues, better to not include it, I dropped the commit.